crdtree/src/CRDTree.ts

```diff
@@ -18,12 +18,13 @@ export class CRDTree<T = any> implements ICRDTree<T> {
 		this.state = new State<T>(from);
 	}
 
-	private makeChange(action: FrontendAction): void {
+	private makeChange(action: FrontendAction, ref?: BranchID): void {
 		const backendChanges = this.insertChanges([{
 			action: action,
 			clock: this.state.next(),
 			pid: this.pid,
 			dep: this.state.latest(),
```

Comment:
I think this is wrong if you have passed a ref. Shouldn't this be the latest on branch `ref`?

Reply:
This is only used for fork to make a change on a different branch. It's not possible at the API level for someone to make a change on a branch they have not checked out. I will throw in an assert fork though